diff --git a/build.psm1 b/build.psm1
index a8b2be31d..bc0f2c9ca 100644
--- a/build.psm1
+++ b/build.psm1
@@ -425,7 +425,7 @@ function Start-PSBuild {
                      "linux-arm",
                      "linux-arm64",
                      "linux-x64",
-                    "linux-ppc64el",
+                    "linux-ppc64le",
                      "osx-arm64",
                      "osx-x64",
                      "win-arm",
@@ -613,10 +613,10 @@ Fix steps:
     }
 
     $Arguments += "/property:AppDeployment=$AppDeployment"
-    # As the ReadyToRun package for linux-ppc64el is not available on NuGet,
+    # As the ReadyToRun package for linux-ppc64le is not available on NuGet,
     # we must explicitly disable ReadyToRun compilation for this runtime.
     # This addresses the NETSDK1094 error for this specific platform.
-    if ($Options.Runtime -eq 'linux-ppc64el') {
+    if ($Options.Runtime -eq 'linux-ppc64le') {
         $Arguments += "/property:PublishReadyToRun=false"
         $Arguments += "/property:WarnAsError=false" # Do not treat warnings as errors for ppc64le
         $Arguments += "/property:RunAnalyzers=false" # Disable analyzers for ppc64le
@@ -1229,7 +1229,7 @@ function New-PSOptions {
                      "linux-arm",
                      "linux-arm64",
                      "linux-x64",
-                    "linux-ppc64el",
+                    "linux-ppc64le",
                      "osx-arm64",
                      "osx-x64",
                      "win-arm",
